.github/actions/setup-test-runner/action.yml

```
@@ -0,0 +1,34 @@
+runs:
+  using: composite
+  steps:
+    - uses: ./.github/actions/setup-node-deps
+    - if: inputs.camoufox == 'true'
+      uses: ./.github/actions/install-camoufox
+      with:
+        gh-token: ${{ inputs.gh-token }}
+    - if: inputs.tests-config == 'true'
+      shell: bash
+      run: npx ncp src/Tests/.tests-config.tpl.cjs src/Tests/.tests-config.cjs
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)